provisioning/Samples/service/BulkOperationSample/Parameters.cs

```diff
@@ -9,7 +9,7 @@ namespace BulkOperationSample
     internal class Parameters
     {
         [Option(
-            'p',
+            'c',
             "ProvisioningConnectionString",
             Required = false,
             HelpText = "The primary connection string of device provisioning service. Not required when the PROVISIONING_CONNECTION_STRING environment variable is set.")]
```

Comment:
We can now strike the word "primary" as it doesn't matter which they give us and we don't need to give a hint as to why the alias is `-p`.